server/tests/api/object-storage/videos.ts

```diff
@@ -107,17 +107,18 @@ async function checkFiles (options: {
 }
 
 function runTestSuite (options: {
+  fixture?: string
```

Comment:
I don't understand why you removed `maxUploadPart` and instead use a big fixture

Reply:
Since lib-storage has a minimum limit at 5 mb for the parts, so the old test will make lib-storage to through an error.

https://github.com/aws/aws-sdk-js-v3/blob/4d9f581de6633dd0a7cde71b094fdb07fe2910dc/lib/lib-storage/src/Upload.ts#L27

Reply:
Okay then could you try to generate the big video file to not increase repo side? Like https://github.com/Chocobozzz/PeerTube/blob/develop/server/tests/shared/generate.ts#L17